### ISSUE TITLE
[Skyworth 40S3B android 4.4] I use 4K Alt Mode, and always "App youtube Stopped" after some minute use

### ISSUE THREAD

**Issue by tranminhthach** [yuliskov/SmartYouTubeTV#211]
I use 4K Alt Mode, and always "App youtube Stopped" after some minute use

**Comment by yuliskov** [comment]
@tranminhthach hi, have you noted when this happens? Is it happens when you playing videos or when navigating through the content?

**Comment by yuliskov** [comment]
duplicate of #119

**Comment by tranminhthach** [comment]
it only happen when i were playing a video, when i were navigating, it run so good and not problem

**Comment by yuliskov** [comment]
@tranminhthach one more question. How much ram does your tv have?

**Comment by tranminhthach** [comment]
1GB Ram

**Comment by yuliskov** [comment]
@tranminhthach ok. Try to test this version.
https://github.com/yuliskov/SmartYouTubeTV/releases/download/6.15.48/SYTV_Orig_v6.15.48_r.apk

**Comment by tranminhthach** [comment]
Ok. Thanks i tried test this version but until like after that, thanks for supportting of you, i will follow version update new of SYTU.

**Comment by tranminhthach** [comment]
Mode 1080 is run good, but is very slow, In 4k Mode, it run very fast, and it wasn't lag but stopped when seen a video. I think you can fix mode 1080 Mode to run better. Thanks so much for supportting this

**Comment by yuliskov** [comment]
@tranminhthach thanks for the reply. So you didn't notify changes on 4K/4K Alt modes compared to the previous builds?

**Comment by tranminhthach** [comment]
Yes, I tried it, I tried switching between the two modes, But the application is stopped after a few minutes of use

**Comment by yuliskov** [comment]
@tranminhthach unfortunately, I still can't find a source of the problem. The internal logging routine, that has been implemented in my app, doesn't send any info regarding app's crash ( I wonder why this happens). So, I desperately need to take logs when app get crashed. Could you tell me, do you have root access to your tv? In this case it is possible to take system logs.

**Comment by yuliskov** [comment]
I've added logging capabilities to the latest beta version. Pls, check out tweaks menu on the main boot screen.
